app/views/colors/show.html.erb

```diff
@@ -4,6 +4,36 @@
   <p>Nickname: <%= @color.nickname %></p>
 <% end %>
 
+<%= form_for [ @color, @review ] do |f| %>
+<!--
+  <% if @review.errors.any? %>
+    <div id="error_explanation">
+      <h3><%= pluralize(@review.errors.count, "error") %> prohibited this review from being saved:</h3>
+      <ul>
+        <% @review.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %> -->
```

Comment:
Commented out code! If we're not ready to use code it's a good idea not to pull it into master.